lib/core/presentation/widgets/event/event_post_card_widget.dart

```diff
@@ -152,6 +175,12 @@ class EventPostCard extends StatelessWidget {
   }
 
   Container _buildEventPhoto() {
+    final imageUrl = eventPhoto != null
+        ? ImageUtils.generateUrl(
+            file: eventPhoto,
+            imageConfig: ImageConfig.eventPhoto,
+          )
+        : '${AppConfig.assetPrefix}/assets/images/no_photo_event.png';
```

Comment:
when ever you add new image or icon, use ./build_runner.sh, fluttergen will generate assets for you, 
the usage will be like 
`Assets.images.noPhotoEvent.image()`
You can check the usage with other svg

Reply:
Yep, but this one is using the dynamic `no_photo_event.png` load from our assets server, so don't need store that png in app (increase size)